application-xpoll-ui/src/main/resources/XPoll/XPollClass.xml

```diff
@@ -47,9 +47,11 @@
     <nameField/>
     <validationScript/>
     <description>
+      <contenttype>---</contenttype>
```

Comment:
This kind of change, and many others, represent formatting changes introduced in from an XWiki version to another. In order to avoid them in commits, you should work on the minimal supported XWiki version, in this case XWiki 11.10. 
If you happen to work on the right version, then you could commit these changes separately, to isolate the solution you propose from anything else that could be refactoring or formatting, as much as possible.